platform/podjs/src/browserPod.ts

```diff
@@ -362,9 +357,23 @@ class BrowserNetwork {
             request.onerror = function () {
                 fetchResponse.error = `Network error`;
                 resolve(fetchResponse);
+                return;
```

Comment:
Does this do anything? It's a return without value at the end of a function, seems redundant.

Reply:
Yeah I thought this was weird as well, but was the only way to stop further code execution (TBH I thought resolve was capable of doing this)

Reply:
Calling `resolve` doesn't return, of course. But my point is - you don't _need_ to return here, since you're at the very end of an anonymous function. I'll eat a broom if this is _actually_ needed.